### PR TITLE
Add documentation for Switch as X

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

Add documentation for Switch as X

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/67878
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_redirects [hunk 1]
```diff
@@ -165,9 +165,11 @@
 /components /integrations
 
 # Renamed integrations
+/integrations/air_pollutants /integrations/air_quality
```
Comment (klaasnicolaas) [inline source/_redirects]: Does this change really belong in the PR?

Comment (frenck) [reply source/_redirects]: Uh no, bit still broken on dev 😁

Comment (frenck) [reply source/_redirects]: Removed 👍

File: source/_integrations/switch_as_x.markdown [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ha_domain: switch_as_x
+ha_config_flow: true
+---
+
+The Switch as X integrations lets you convert any Home Assistant switch into
+a Home Assistant Light, Cover, Fan, Lock, or Siren.
+
+In Home Assistant's world, a wall plug is a switch. And while that is correct
+for a wall plug in general, those plugs are often used with e.g, a light
+fixture or fan. General-purpose relays are similar, as sometimes used for
```
Comment (emontnemery) [inline source/_integrations/switch_as_x.markdown]:
```suggestion
fixture or a fan. General-purpose relays are similar, as they are sometimes used for
```

**Review by emontnemery — APPROVED** [review]

**Pull request merged**